### PR TITLE
Update Language samples.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: language/analyze.js [hunk 1]
```diff
@@ -1,304 +1,220 @@
...
-  document.annotate(config, function (err, result, apiResponse) {
-    if (err) {
-      return callback(err);
-    }
-
-    console.log('Done analyzing syntax');
-    return callback(null, apiResponse);
-  });
+  // Detects syntax in the document
+  return document.detectSyntax()
```
Comment (ace-n) [inline language/analyze.js]: Nit: this seems kinda repetitive, but I'm not sure whether/how to avoid this

File: language/analyze.js [hunk 2]
```diff
@@ -1,304 +1,220 @@
...
-  analyzeSentimentFromFile: analyzeSentimentFromFile,
-  analyzeEntitiesFromString: analyzeEntitiesFromString,
-  analyzeEntitiesFromFile: analyzeEntitiesFromFile,
-  analyzeSyntaxFromString: analyzeSyntaxFromString,
-  analyzeSyntaxFromFile: analyzeSyntaxFromFile,
-  main: function (args) {
+const cli = require(`yargs`);
+
+const program = module.exports = {
+  analyzeSentimentOfText: analyzeSentimentOfText,
```
Comment (ace-n) [inline language/analyze.js]: With ES6, do we need to do `analyze...Text: analyze...Text`? (Can't we just do `analyze...Text`?)

File: language/system-test/analyze.test.js [hunk 3]
```diff
@@ -13,103 +13,66 @@
...
-        assert.equal(err, null);
-        assert.equal(typeof sentiment, 'object');
-        assert.equal(typeof sentiment.polarity, 'number');
-        assert.equal(typeof sentiment.magnitude, 'number');
-        done();
-      });
-    });
+  it(`should analyze sentiment in text`, () => {
+    const output = run(`${cmd} sentimentOfText "${text}"`, cwd);
+    assert.equal(typeof parseFloat(output), `number`);
```
Comment (ace-n) [inline language/system-test/analyze.test.js]: What exactly is the point of this? (To make sure `output` is a number?)

File: language/test/analyze.test.js [hunk 4]
```diff
@@ -1,368 +1,29 @@
...
-      var program = getSample().program;
-
-      sinon.stub(program, 'analyzeSyntaxFromFile');
-      program.main(['syntaxFromFile', bucketName, fileName]);
-      assert.equal(program.analyzeSyntaxFromFile.calledOnce, true);
-      assert.deepEqual(program.analyzeSyntaxFromFile.firstCall.args.slice(0, -1), [bucketName, fileName, options]);
-    });
+describe(`language:analyze`, () => {
+  it(`should have expected exports`, () => {
+    assert.equal(typeof program.analyzeSentimentOfText, `function`);
```
Comment (ace-n) [inline language/test/analyze.test.js]: Is this necessary? (Won't it "halt and catch fire" if this is missing, or is it better to explicitly check this to save debugging time?)

File: language/analyze.js (hunk 2 above)
Comment (jmdobry) [inline language/analyze.js]: good point.

File: language/test/analyze.test.js (hunk 4 above)
Comment (jmdobry) [inline language/test/analyze.test.js]: Yeah, I guess I could just delete this file.

File: language/analyze.js (hunk 1 above)
Comment (jmdobry) [inline language/analyze.js]: Ack

**Pull request merged**